ntbs-service/Models/Validations/ValidationAttributes.cs

```diff
@@ -44,4 +45,47 @@ private bool isTruthy(object value)
             return value != null && (bool)value;
         }
     }
+
+    public class ValidPartialDateCanConvertToDatetimeAttribute : ValidationAttribute
+    {
+        protected override ValidationResult IsValid(object value, ValidationContext validationContext)
+        {
+            PartialDate valueCopy = (PartialDate) value;
+            if(valueCopy==null || valueCopy.IsEmpty()) {

```

Comment:
This line is needed (and needed on the copy of the value that casts it to PartialDate) as the object is initialised with null values a lot of the time so a simple if(value == null) doesn't work